include/sway/tree/container.h

```diff
@@ -0,0 +1,143 @@
+#ifndef _SWAY_CONTAINER_H
+#define _SWAY_CONTAINER_H
+#include <stdint.h>
+#include <sys/types.h>
+#include <wlr/types/wlr_box.h>
+#include <wlr/types/wlr_surface.h>
+#include "list.h"
+
+extern struct sway_container root_container;
+
+struct sway_view;
+struct sway_seat;
+
+/**
+ * Different kinds of containers.
+ *
+ * This enum is in order. A container will never be inside of a container below
+ * it on this list.
+ */
+enum sway_container_type {
+	C_ROOT,
+	C_OUTPUT,
+	C_WORKSPACE,
+	C_CONTAINER,
+	C_VIEW,
+
+	C_TYPES,
```

Comment:
Any reason why these aren't prefixed?

Reply:
They get typed a lot so it's more ergonomic. Since sway isn't a library I prefer it this way.